yarn.lock

```diff
@@ -1499,7 +1499,6 @@ fsevents@^1.2.3:
   dependencies:
     bindings "^1.5.0"
     nan "^2.12.1"
-    node-pre-gyp "*"
```

Comment:
❓

Reply:
Don't know, I ran `yarn`

Reply:
Generating a new lock

Reply:
@ikatyang I deleted the lock file, and ran `yarn` again, still missing this line, maybe because yarn version? Mine is `1.21.1`

Reply:
I got the same result as well (v1.17.3), the weird thing is that you'll get a lot of diffs if you remove your `node_modules` and run `yarn` again.

Reply:
I restored this missing line, by editing the lock file